doc/arch/adr-002-handle-race-conditions-in-govuk-index.md

```diff
@@ -0,0 +1,105 @@
+# Decision record: use versioning to handle race conditions when updating the govuk index
```

Comment:
Both seem valid to me.

I'd lean towards including the solution in the title because if you're interested in understanding why something is the way it is, you should be able to scan the list of ADRs and see if there's anything relevant, but you may not know the original problem beforehand.